pages/etherpad/[[...roomId]].js

```diff
@@ -148,6 +149,25 @@ export default function Etherpad() {
         return room;
     }, [auth, matrix, matrixClient, etherpad]);
 
+    /**
+     * Creates an Etherpad and navigates to it based on the provided name, visibility, and password.
+     *
+     * @param {string} name - The name of the Etherpad to create.
+     * @param {string} visibility - The visibility setting for the Etherpad. Expects 'private' or 'public'.
+     * @param {string} password - The password for accessing the Etherpad (if private).
+     * @returns {Promise<boolean|void>} - A Promise that resolves to a 'false' indicating the success of pad creation or navigates to the Etherpad's URL.
```

Comment:
This I don't get...

> A Promise that resolves to a 'false' indicating the success of pad creation

`false` indicates the successful creation?

Reply:
oops that's the wrong way around...